pages/Home.tsx

```diff
@@ -167,9 +167,10 @@ export function Home() {
                       {balanceDisplayMode === "sats" &&
                         getFiatAmount?.(Math.floor(balance.balance / 1000))}
                       {balanceDisplayMode === "fiat" &&
-                        new Intl.NumberFormat().format(
-                          Math.floor(balance.balance / 1000),
-                        ) + " sats"}
+                        "₿ " +
```

Comment:
I don't think we can so easily change this, can you move it to a new PR and we need to do it properly like in the hub (unit display in settings, consistently apply everywhere, not just on the home screen)